src/oc/web/actions/reaction.cljs

```diff
@@ -0,0 +1,72 @@
+(ns oc.web.actions.reaction
+  (:require [oc.web.api :as api]
+            [oc.web.lib.jwt :as jwt]
+            [oc.web.router :as router]
+            [oc.web.dispatcher :as dis]
+            [oc.web.lib.utils :as utils]
+            [oc.web.lib.json :refer (json->cljs)]))
+
+(defn react-from-picker [activity-data emoji]
+  (dis/dispatch! [:handle-reaction-to-entry activity-data
+   {:reaction emoji :count 1 :reacted true :links [] :authors []}])
+  (when (and emoji
```

Comment:
is there a case where the emoji is nil/false?

Reply:
The point is that we need to use native emojis, the picker pass to use a js object containing not only the emoji char but some information about it and some times `emoji.native` can be null.
That's why i added that check, adding a comment.